code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -819,6 +878,11 @@
 			dat += text("[module.emag]: <B>Activated</B><BR>")
 		else
 			dat += text("[module.emag]: <A HREF=?src=\ref[src];act=\ref[module.emag]>Activate</A><BR>")
+	if(malfAImodule)
+		if(activated(malfAImodule))

```

Comment:
Shouldn't this be `activated(module.malfAImodule)`? `src.malfAImodule` is an integer.